packages/terra-consumer-nav/src/components/nav-help/NavHelpContent.jsx

```diff
@@ -44,12 +44,12 @@ class NavHelpContent extends React.Component {
       let contentElement;
       const isOpen = this.state.togglers[index];
       const toggleIcon = isOpen ? <IconChevronUp /> : <IconChevronDown />;
+      const helpItemStyle = cx('help-item', index > 0 && 'help-item-border');
```

Comment:
can we not apply the border rule with CSS rather than logically adding another class name?

Reply:
Also the correct way to do conditional class names is
```js
cx({ 'help-item-border': index  > 0 })
```
https://www.npmjs.com/package/classnames#usage